int64 numKernels
int64 numRows
float64 radius
float64 height
float64[] globalPos
float64 variance
float64 numPolicies